packages/eslint-plugin/ROADMAP.md

```diff
@@ -1,9 +1,9 @@
 # Roadmap
 
-✅ (27) = done  
-🌟 (79) = in ESLint core  
-🔌 (33) = in another plugin  
-🌓 (16) = implementations differ or ESLint version is missing functionality  
+✅ (27) = done
+🌟 (79) = in ESLint core
+🔌 (33) = in another plugin
+🌓 (16) = implementations differ or ESLint version is missing functionality
```

Comment:
Can you add these trailing spaces back please? They preserve the line breaks.

Actually, could you replace them with `<br />` so it’s more explicit?